hail/src/main/scala/is/hail/expr/ir/InferPType.scala

```diff
@@ -120,18 +120,18 @@ object InferPType {
         throw new RuntimeException(s"error while inferring IR:\n${Pretty(ir)}", e)
     }
     VisitIR(ir) { case (node: IR) =>
-      if (node._pType2 == null)
+      if (node._pType == null)
```

Comment:
structure is: _pType is only used for null checks and as a setter. pType for getters.